main.py

```diff
@@ -1,10 +1,38 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for key, value in data.items():
```

Comment:
```suggestion
    for nickname, data in data.items():
```
these names will be more descriptive